### PR TITLE
Fix python3 setup

### PR DESCRIPTION
**Pull request by andfasano**

### CODE REVIEW

**Comment by andfasano** [comment]
cc @elfosardo

File: 01_install_requirements.sh [hunk 1]
```diff
@@ -54,11 +54,10 @@ fi
 sudo dnf -y install python39
 sudo alternatives --set python /usr/bin/python3.9
 sudo alternatives --set python3 /usr/bin/python3.9
-sudo update-alternatives --install /usr/bin/pip3 pip3 /usr/bin/pip3.9 1
-sudo pip3 install ansible=="${ANSIBLE_VERSION}"
+sudo python -m pip install ansible=="${ANSIBLE_VERSION}"
 # Also need the 3.9 version of netaddr for ansible.netcommon
 # and lxml for the pyxpath script
```
Comment (elfosardo) [inline 01_install_requirements.sh:59]: maybe also mention the selinux module ?

**Review by elfosardo — CHANGES_REQUESTED** [review]
we should also change the entry in metallb/run_e2e.sh and the "yq install message instruction" in metal3-dev/local-bmo.sh and metal3-dev/local-capbm.sh

**Comment by andfasano** [comment]
> we should also change the entry in metallb/run_e2e.sh and the "yq install message instruction" in metal3-dev/local-bmo.sh and metal3-dev/local-capbm.sh

Ok for metal-3-dev, but metallb has its own owners (@fedepaol)

**Review by elfosardo — APPROVED** [review]

**Pull request closed without merging**